backend/src/bundles/users/user.controller.ts

```diff
@@ -37,6 +40,14 @@ class UserController extends Controller {
             method: 'GET',
             handler: () => this.findAll(),
         });
+
+        this.addRoute({
+            path: `${UsersApiPath.ROOT}`,
+            method: 'PUT',
+            handler: (options) => {
+                return this.update(options as UpdateRequestDto);
```

Comment:
No backticks needed here

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/98/commits/3606ef6d2bc9fa2faa9acfe9d665af47df196d7a](url)